inst/private/python_header.py

```diff
@@ -43,6 +43,7 @@ def echo_exception_stdout(mystr):
     import struct
     import codecs
     import xml.etree.ElementTree as ET
+    import mpmath
```

Comment:
I'm not sure we want this change: I don't think there is a significant performance penalty to importing the same thing repeatedly.  And I think functions should not be using mpmath directly (where possible).